README.md

```diff
@@ -16,8 +16,9 @@ For further developer and user documentation please visit [the wiki](https://git
 ##### Minimum PHP Version
 * PHP >= 5.4
 
-##### Minimum ownCloud Version
-* >= 8.1
+##### Minimum ownCloud / Nextcloud Version
```

Comment:
Don't we want to remove ownCloud completely as we have done it in the other apps?